js/about/extensions.js

```diff
@@ -9,13 +9,51 @@ const ImmutableComponent = require('../components/immutableComponent')
 const messages = require('../constants/messages')
 const aboutActions = require('./aboutActions')
 
+const {StyleSheet, css} = require('aphrodite/no-important')
+const globalStyles = require('../../app/renderer/components/styles/global')
+const commonStyles = require('../../app/renderer/components/styles/commonStyles')
+
 const ipc = window.chrome.ipcRenderer
 
 // Stylesheets
-require('../../less/about/itemList.less')
-require('../../less/about/extensions.less')
+require('../../less/about/common.less')
 require('../../node_modules/font-awesome/css/font-awesome.css')
 
+const styles = StyleSheet.create({
```

Comment:
I put that here bearing in mind that on HTML files inlined styles are placed before body tag. Moving these to the bottom is fine for me. CC @bbondy and @bsclifton for comments. I think we do not have the rule on the placement (Sorry if I missed it).